src/base/constants.jl

```diff
@@ -9,7 +9,8 @@ const _constants = (
     tolerance = 1e-6,
     sampling_keep_iters = 100,
     sampling_size = 1000,
-    exchange_suffixes = ["_e", "[e]", "(e)"],
+    strict_inequality_tolerance = 1,
```

Comment:
naming: perhaps add something about what purpose this serves, like "loopless_inequality_tolerance" or "slack_variable_bound_tolerance" or something?